test/test_helper.rb

```diff
@@ -39,7 +39,8 @@ class Application < Rails::Application
       config.secret_key_base = "stub"
       config.eager_load = false
     end
-    ActiveRecord::Base.establish_connection(:adapter => "sqlite3", :database => ":memory:")
+    File.unlink("db/testdb.sqlite3")
```

Comment:
I am not entirely certain we want to do the unlinking of the file on each start of the test run.

Ideally, subsequent test runs should "overlap safely" on the same database (for example, this is how the rails test suite works)...

Alternatively, I would advise using a completely random file via `Tempfile` in a local tmpdir http://ruby-doc.org/stdlib-2.2.3/libdoc/tempfile/rdoc/Tempfile.html and then have a `rake db:test:sweep` that clears that directory